pandas/core/frame.py

```diff
@@ -1144,32 +1143,58 @@ def to_gbq(self, destination_table, project_id, chunksize=10000,
         Parameters
         ----------
         dataframe : DataFrame
-            DataFrame to be written
+            DataFrame to be written to Google BigQuery.
         destination_table : string
-            Name of table to be written, in the form 'dataset.tablename'
+            Name of table to be written, in the form 'dataset.tablename'.
         project_id : str
             Google BigQuery Account project ID.
         chunksize : int (default 10000)
```

Comment:
"int (default 10000)" -> "int, default 10000"

(and same comment for other similar cases below)